connector-j-5/pom.xml

```diff
@@ -19,6 +19,26 @@
     SSL certificates manually.
   </description>
 
+  <dependencyManagement>
```

Comment:
Rather than repeat these 3 in each component, is there any reason not to put them all in the parent pom?

Reply:
That would be a better idea. I wasn't sure if the required versions for the components would be different, but since they're all pretty much the same, I can move this to the parent pom.